network/src/noise/handshake.rs

```diff
@@ -248,15 +258,42 @@ impl NoiseUpgrader {
             .map_err(|e| io::Error::new(io::ErrorKind::Other, e))?;
 
         // send the first handshake message
+        sl_info!(
+            network_log(network_events::NOISE_UPGRADE, &self.network_context)
+                .data("origin", "client")
+                .data("step", "writing_data")
+                .log(format!(
+                    "{} noise client writing_data: {}",
+                    self.network_context, remote_public_key
+                ))
+        );
         socket.write_all(&client_message).await?;
         socket.flush().await?;
 
         // receive the server's response (<- e, ee, se)
+        sl_info!(
+            network_log(network_events::NOISE_UPGRADE, &self.network_context)
+                .data("origin", "client")
+                .data("step", "reading_data")
+                .log(format!(
+                    "{} noise client reading_data: {}",
+                    self.network_context, remote_public_key
```

Comment:
same here, why not log the actual server_response after reading it? remote_public_key here is the expected remote public key not the one read

Reply:
this is before the server response, we want to make sure we're getting this far.

Reply:
I would suggest modifying this log then to reflect that, right now it sounds like you're reading that key, when that's not what's happening : o maybe "waiting to read server response"

Reply:
we are in the process of reading the data :P I think they both mean the same thing... if we wanted to be after the event, we could say "server response: ..."